virt_who/register.py

```diff
@@ -11,7 +11,9 @@ def qa_register(self, ssh):
         serverurl = "--serverurl={0}".format(deploy.register.serverurl)
         baseurl = "--baseurl={0}".format(deploy.register.baseurl)
         account = "--username={0} --password={1}".format(deploy.register.username, deploy.register.password)
-        employee_sku = deploy.register.employee_sku
+        employee_sku_phys = deploy.register.employee_sku_phys
+        employee_sku_virt = deploy.register.employee_sku_virt
```

Comment:
`deploy.register.employee_sku_phys` and `deploy.register.employee_sku_virt` should be defined in settings.py

Reply:
have defined and committed it.